docs/admin/install/helm-install/google-gke.md

```diff
@@ -3,6 +3,186 @@ tags:
+| scim              | 1.0      | 1.0GB | N/A        | 64 Bit         | No                                 |
+| config - job      | 0.5      | 0.5GB | N/A        | 64 Bit         | Yes on fresh installs              |
+| persistence - job | 0.5      | 0.5GB | N/A        | 64 Bit         | Yes on fresh installs              |
+| client-api        | 1        | 0.4GB | N/A        | 64 Bit         | No                                 |
+| nginx             | 1        | 1GB   | N/A        | 64 Bit         | Yes if not ALB                     |
+| auth-key-rotation | 0.3      | 0.3GB | N/A        | 64 Bit         | No [Strongly recommended]          |
+| config-api        | 0.5      | 0.5GB | N/A        | 64 Bit         | No                                 |
+
+Releases of images are in style 1.0.0-beta.0, 1.0.0-0
+
+## Initial Setup
+
+1.  If you are using Cloud Shell, you can skip to step 4.
+
+2.  Install [gcloud](https://cloud.google.com/sdk/docs/quickstarts)
+    
+3.  Install kubectl using `gcloud components install kubectl` command
+    
+4.  Create cluster using a command such as the following example:
+
+    ```  
+    gcloud container clusters create CLUSTER_NAME --num-nodes 2 --machine-type e2-highcpu-8 --zone ZONE_NAME
+    ```
+    where `CLUSTER_NAME` is the name you choose for the cluster and `ZONE_NAME` is the name of Availability Zone, for example: us-west1-a (https://cloud.google.com/compute/docs/regions-zones/) where the cluster resources live in.
+
+5.  Install [Helm3](https://helm.sh/docs/intro/install/)    
+
+6.  Create `jans` namespace where our resources will reside
+    ```
+    kubectl create namespace jans
+    ```
+
+## Jans Installation using Helm
+1.  Install [Nginx-Ingress](https://github.com/kubernetes/ingress-nginx), if you are not using Istio ingress
+    
+      ```
+      helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+      helm repo add stable https://charts.helm.sh/stable
+      helm repo update
+      helm install nginx ingress-nginx/ingress-nginx
+      ```
+
+2.  Create override.yaml file and add changes as per your desired configuration:
+
+    - FQDN/domain not registered:
+    
+        
+        Get the Loadbalancer IP: 
+        ```
+        kubectl get svc nginx-ingress-nginx-controller --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+        ```
+
+      
+        
+        Add this to override.yaml:
+
+        ```yaml
+        global:
+            lbIp: #Add IP from previous command
+            isFqdnRegistered: false
+        ```
+
+    - FQDN/domain registered:
+    
+        If the FQDN for jans i.e `demoexample.jans.org` is registered and globally resolvable, you have to map it to the loadbalancer IP created in the previous step by Nginx-Ingress.
+        More details in this [guide](https://medium.com/@kungusamuel90/custom-domain-name-mapping-for-k8s-on-gcp-4dc263b2dabe).
```

Comment:
I don't think we should add this. Please remove it. The method normally is to add the record via IaC and I think this is rather intuitive.

Reply:
Yeah the guide link might be unnecessary and can be removed.
However, I think the user might forget that he must map his domain to the LB IP in case of GKE, LB address in case of EKS..etc
I am thinking of leaving the first line. What do you think?